libs/api/domains/driving-license/src/lib/graphql/applicationEligibilityRequirement.model.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+
+@ObjectType()
+export class ApplicationEligibilityRequirement {
+  @Field(() => ID)
+  key!: string
+
+  @Field(() => Boolean)
```

Comment:
Graphql generation should resolve basetypes automagically.
There is no need for a type resolution function in these cases